app/docs/0.10.x/proxy.md

```diff
@@ -460,25 +447,25 @@ HEAD /resource HTTP/1.1
 Host:
 ```
 
-But would not match a `POST` or a `DELETE` request. This allows for much more 
+But would not match a `POST` or `DELETE` request. This allows for much more 
 granularity when configuring APIs and Plugins. For example, one could imagine 
-two APIs pointing to the same upstream service: one allowing for `GET` requests
-and unprotected, and a second one filtering out `POST` requests and applying 
-authentication or rate limiting plugins on such requests.
+two APIs pointing to the same upstream service; one API allowing unlimited unauthenticated `GET` requests, and a second API allowing only authenticated and rate-limited `POST` requests (by applying 
+the authentication and rate limiting plugins to such requests).
 
 [Back to TOC](#table-of-contents)
 
 ### Routing priorities
 
-It is now clear that an API may define matching rules based on its `hosts`, 
-`uris`, and `methods` fields. It is also clear that for Kong to match an
+An API may define matching rules based on its `hosts`, 
+`uris`, and `methods` fields. For Kong to match an
```

Comment:
please ensure consistent use of oxford comma (line 95 is an example of alternative, pick and stick with one :) )